### PR TITLE
add github actions

### PR DESCRIPTION
**Pull request by kcpevey**

### CODE REVIEW

**Comment by kcpevey** [comment]
Ready for review. This switches over to the github actions CI system. I've added a CI step to run the `download_sample_data.sh` and a simple test to ensure that the sample data does indeed exist.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      - name: download data
+        shell: bash -l {0}
+        run: |
+          ./download_sample_data.sh
+          cd sample_data
+          tar -xzf *
+          ls
+      - name: pytest
+        shell: bash -l {0}
+        run: pytest --cov-report term-missing --cov=lsst_dashboard tests/
```
Comment (timothydmorton) [inline .github/workflows/main.yml]: can we add `-v` to the pytest run?

**Comment by timothydmorton** [comment]
Looks good to me; I'm happy if you are, @dharhas-- this is my introduction to github actions, which looks great.  When this is in, we could then add some more meaningful tests.

**Comment by dharhas** [comment]
LGTM. @kcpevey go ahead and add the -v flag and we can merge

**Pull request merged**